### PR TITLE
some work on archetypes

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2024-08-28-blop/index.qmd [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# - Note "/" between year/month/day
+socialImg: blog/2019/06/04/post-template/name-of-image.png
+socialAlt: "Alternative description of the image"
+# the text below is for populating the "share on Twitter" button
+# if deleted, the title of the post will be used
+social: "A post about blabla by @username@server.extension!"
+format: hugo-md
+---
+
+This is the R Markdown (.Rmd) template for a blog post or tech note.
```
Comment (steffilazerte) [inline content/blog/2024-08-28-blop/index.qmd:31]:
```suggestion
This is the Quarto (.qmd) template for a blog post or tech note. 
```

File: content/blog/2024-08-28-blop/index.qmd [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+---
+
+This is the R Markdown (.Rmd) template for a blog post or tech note. 
+To generate your post with just Markdown (.md), use that template instead.
+
+Throughout this template, including the YAML, 
+you should change "post-template" to the slug of your post, 
+and "2019-06-04" to your publication date.
+
+Save this file under /content/blog/YYYY-MM-DD-slug/index.Rmd in the local copy of your roweb3 fork.
```
Comment (steffilazerte) [inline content/blog/2024-08-28-blop/index.qmd:38]:
```suggestion
Save this file under /content/blog/YYYY-MM-DD-slug/index.qmd in the local copy of your roweb3 fork.
```

File: content/blog/2024-08-28-blop/index.qmd [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+The chunk below ensures your plots will be inserted in the same folder as your Rmd so you can submit the bundle (.Rmd, .md, images) to roweb3.
```
Comment (steffilazerte) [inline content/blog/2024-08-28-blop/index.qmd:40]:
```suggestion
The chunk below ensures your plots will be inserted in the same folder as your qmd so you can submit the bundle (.qmd, .md, images) to roweb3.
```

**Review by steffilazerte — COMMENTED** [review]
Looks good to me, I would just change a bit of the example language to be explicit about qmd vs. Rmd.

**Comment by maelle** [comment]
Please tell me when anyone uses this archetype!

**Comment by steffilazerte** [comment]
Will do!

**Pull request merged**